client/client.go

```diff
@@ -58,7 +58,7 @@ import (
 )
 
 // DefaultVersion is the version of the current stable API
-const DefaultVersion string = "1.23"
+const DefaultVersion string = "1.25"
```

Comment:
Yes, I was thinking of that, and wondering why we didn't do that. Is it ok in this case to import one into the other? (possibly these packages were meant to be kept separate)